pepper-apis/src/main/java/org/broadinstitute/ddp/model/activity/instance/validation/DecimalRangeRule.java

```diff
@@ -0,0 +1,59 @@
+package org.broadinstitute.ddp.model.activity.instance.validation;
+
+import com.google.gson.annotations.SerializedName;
+import org.broadinstitute.ddp.model.activity.instance.answer.DecimalAnswer;
+import org.broadinstitute.ddp.model.activity.instance.question.Question;
+import org.broadinstitute.ddp.model.activity.types.RuleType;
+
+import java.math.BigDecimal;
+
+/**
+ * A validation rule that checks numeric integer value is within an optional min/max range, inclusive.
+ */
+public class DecimalRangeRule extends Rule<DecimalAnswer> {
```

Comment:
Same question as above about having separate range rules. I can understand why we might have separate Java implementations (I think) but not sure it buys us anything in the API.

Reply:
The Integer range validation has integer ranges- that's about the only limitation I can see. If we're ok with that we can I don't see any reason we can't reuse the validation (at least, API-wise)

Reply:
If we're going to reuse the validation, why we're not going to just replace the Numeric type from Integer to Decimal? This also will work. I don't like half/solutions. It will look less clear from my POV. If we do what you offer, we will have two hierarchies of the classes: for Numeric and for Decimal. But Decimal hierarchy will partially reuse Numeric's one.
I guess there are just two options to keep the code readable:
1. Completely replace the type used in Numeric (Integer -> Decimal) and have only one question type - Numeric based on BigDecimal type
2. Keep two independent hierarchies of the classes (like in this PR).

I would say that both solutions are fine. Should we discuss this?